module/VuFind/src/VuFind/Db/Entity/LoginToken.php

```diff
@@ -0,0 +1,356 @@
+<?php
+
+/**
+ * Entity model for access_token table
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ */
+
+namespace VuFind\Db\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Entity model for login_token table
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ *
+ * @ORM\Table(name="login_token")
+ * @ORM\Entity
+ */
+class LoginToken implements LoginTokenEntityInterface
+{
+    /**
+     * Unique ID.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="id",
+     *          type="integer",
+     *          nullable=false
+     * )
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="NONE")
+     */
+    protected $id;
+
+    /**
+     * User ID.
+     *
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="VuFind\Db\Entity\User")
+     * @ORM\JoinColumns({
+     * @ORM\JoinColumn(name="user_id",
+     *              referencedColumnName="id")
+     * })
+     */
+    protected $user;
+
+    /**
+     * Token.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="token",
+     *             type="string",
+     *             length=255,
+     *             nullable=false
+     * )
+     */
+    protected $token;
+
+    /**
+     * Series.
+     *
+     * @var string
+     *
+     * @ORM\Column(name="series",
+     *             type="string",
+     *             length=255,
+     *             nullable=false
+     * )
+     */
+    protected $series;
+
+    /**
+     * Last login date.
+     *
+     * @var \DateTime
+     *
+     * @ORM\Column(name="last_login",
+     *          type="datetime",
+     *          nullable=false
+     * )
+     */
+    protected $lastLogin = '2000-01-01 00:00:00';
```

Comment:
I'm sure you copied this from somewhere else, and I've seen this in other entity classes, but I think it's something we need to fix: the default value is set as a string, but the type of the property is DateTime, so this is technically wrong. I think the solution is to add a constructor to the class and initialize the property there -- e.g.:

```
public function __construct()
{
    $this->lastLogin = DateTime::createFromFormat('Y-m-d H:i:s', '2000-01-01 00:00:00');
}
```

It might be a good idea to open a follow-up PR after this one to fix the DateTime properties in other entity classes for consistency.